style.css

```diff
@@ -0,0 +1,659 @@
+  }
+
+  .main {
+    padding-left: 0;
+    padding-top: 0;
+    padding-right: 0;
+  }
+
+  header {
+    display: block;
+    background-color: #272a31;
+  }
+
+  .lists {
+    display: flex;
+    justify-content: end;
+    list-style-type: none;
+    color: #fff;
+    margin-right: 13%;
+    gap: 2.76%;
+    padding: 10px;
+    text-align: center;
+    margin-top: 0;
+    margin-bottom: 0;
+  }
+
+  .social-media {
+    color: #fff;
+    text-decoration: none;
+  }
+
+  .nav {
+    display: none;
+  }
+
+  .nav-desk {
+    display: flex;
+    justify-content: space-between;
+    padding-right: 13%;
+    padding-left: 13%;
+    align-items: center;
+    margin-top: 0;
+    background-color: #d3d3d3;
+  }
+
+  .logo-desk > img {
+    width: 90px;
+    height: 80px;
+  }
+
+  .navi-menu-desk {
+    display: flex;
+    gap: 30px;
+  }
+
+  .navi-link-desk {
+    text-decoration: none;
+    color: #272a31;
+    padding: 10px;
+  }
+
+  .home,
+  .sponsor {
+    color: #ec5242;
+  }
+
+  .sponsor {
+    border: 4px solid #ec5242;
+  }
+
+  .partners-icon {
+    transition: all 0.5s ease-in-out;
+    display: inline;
+  }
+
+  .partners-icon2 {
+    transition: all 0.5s ease-in-out;
+  }
+
+  .hello,
+  .main-header,
+  .info,
+  .time {
+    margin-left: 13%;
+    margin-right: 13%;
+  }
+
+  .hello {
+    font-size: 25px;
+    margin-top: 80px;
+  }
+
+  .main-header {
+    font-size: 45px;
+  }
+
+  .info {
+    font-size: 20px;
+    padding: 20px;
+    border: 1px solid white;
+  }
+
+  .interval {
+    font-size: 33px;
+  }
+
+  .place {
+    font-size: 25px;
+  }
+
+  .title {
+    font-size: 33px;
+  }
+
+  .full-information {
+    display: flex;
+    margin-right: 3%;
+    margin-left: 3%;
+    background: url(./images/bgtexture.png);
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 30vh;
+    width: 30vw;
+  }
+
+  .phishing {
+    display: flex;
+    flex-direction: column;
+    margin-top: 20px;
+    background-color: #36393f;
+    border: 1px solid #d3d3d3;
+    padding: 1em;
+    gap: 10px;
+    color: #272a31;
+    transition: all 0.5s ease-in-out;
+  }
+
+  .main1 {
+    background: url(./images/bgtexture.jpeg);
+    background-repeat: no-repeat;
+    background-size: cover;
+    height: 100vh;
+    width: 100vw;
+    border-top: 1px solid #dddbdb;
+  }
+
+  .main-program {
+    display: grid;
+    flex-direction: column;
+    align-items: center;
+    justify-content: center;
+  }
+
+  .main-program h2 {
+    font-size: 2rem;
+    color: #fff;
+    font-family: "Lato", sans-serif;
+    margin-right: 300px;
+  }
+
+  .line {
+    width: 50px;
+    height: 1px;
+    display: flex;
+    align-items: center;
+    justify-content: center;
+    background-color: rgb(237, 76, 18);
+    margin-left: 270px;
+  }
+
+  .program-cards {
+    width: 80%;
+    margin-top: 100px;
+    margin-left: 200px;
+    display: grid;
+    grid-template-columns: repeat(5, 1fr);
+    gap: 1rem;
+  }
```

Comment:
- [ ] [Optional] As it is just one requirement I am adding it as optional, but please take it into consideration as it will improve the overall design of the page, currently the project cards are not centered please see :point_down: 
![image](https://user-images.githubusercontent.com/97521557/211144418-f246a940-0a9b-42a3-879f-d2fe2e37f1a9.png)

you can use 
```
.program-cards {
    width: 80%;
    display: grid;
    grid-template-columns: repeat(5, 1fr);
    gap: 1rem;
    margin: 100px auto;
}
```